client/src/Pages/RideSummary.js

```diff
@@ -0,0 +1,343 @@
+import React, { useState, useEffect } from 'react'
+import styled from 'styled-components'
+import { gql, useQuery } from '@apollo/client'
+// import { graphql } from 'react-apollo'
+import { useParams } from 'react-router-dom'
+import { BsArrowRight } from 'react-icons/bs'
+import {
+  IoLocationSharp,
+  IoShareSocialSharp,
+  IoPersonCircleSharp,
+} from 'react-icons/io5'
+import { IoIosArrowBack } from 'react-icons/io'
+import { AiTwotoneCalendar, AiFillClockCircle } from 'react-icons/ai'
+import moment from 'moment'
+
+//styling
+const SeatsLeftDiv = styled.div`
+  position: absolute;
+  left: 68.36%;
+  right: 0%;
+  top: 12.03%;
+  bottom: 82.04%;
```

Comment:
Your CSS needs a bit of work. Things are going off the screen right now on the ride summary page. I suggest you use `vh` and `vw` instead of percentages for your dimensions. For example, a height of `10 vh` would mean the component is 10 percent as tall as the window. 

In addition, avoid using absolute positioning whenever possible. Using absolute positioning means your components wont change place depending on how big your screen is. Let me know if you need pointers on how to improve your CSS